scan-payment/src/main/java/com/getbouncer/scan/payment/ml/SSDOcr.kt

```diff
@@ -88,11 +88,15 @@ private val FEATURE_MAP_SIZES =
  */
 private val PRIORS = combinePriors()
 
+data class PaymentCardOcrResult(val pan: String?, val name: String?, val expiry: String?)
+
 /**
  * This model performs SSD OCR recognition on a card.
  */
 class SSDOcr private constructor(interpreter: Interpreter) :
-    TensorFlowLiteAnalyzer<SSDOcr.Input, Array<ByteBuffer>, SSDOcr.Prediction, Map<Int, Array<FloatArray>>>(interpreter) {
+    TensorFlowLiteAnalyzer<SSDOcr.SSDOcrInput, Array<ByteBuffer>, SSDOcr.Prediction, Map<Int, Array<FloatArray>>>(interpreter) {
+
+    data class SSDOcrInput(val fullImage: Bitmap, val previewSize: Size, val cardFinder: Rect)
```

Comment:
looks like a rebase gone wrong? this should not exist, it was replaced by `Input` below.